config/locales/en.yml

```diff
@@ -13,5 +13,5 @@ en:
             name:
               taken: " is already registered. If you wish to administer an \n
```

Comment:
Some thoughts on the content of this message:
- It's quite long, I worry people will have trouble understanding it
- It uses `please` - GOV.UK convention is to not
- It's annoying that this has to start with "email" - I think it makes more sense to talk about *accounts* here (I think this is supported by research from other GDS services too). 

Suggested reword:
> You already have a GovWifi account.
> If you want to join the team managing #{org_name}'s GovWifi installation, you need to be invited by someone already on that team.
> If you can't find a member of that team, [contact us](link)

This re-word doesn't take into account technical feasibility. If putting the org name / links in there is hard, could skip some of it.

Reply:
@kjdchapman the reason it has "Email" at the start is because its an ActiveRecord::Base error that applies to the email attribute on the user model. To get rid of email requires doing this as a unique validator and declaring the error message in the model rather than the `yml`.

Reply:
Yeah, sounds like a lot of trouble to sort out - let's skip it.